robot/session_manager_test.go

```diff
@@ -51,6 +50,8 @@ func TestSessionManager(t *testing.T) {
 
 	// Assert that fooSess and barSess can be found with All.
 	allSessions := sm.All()
+	sm.Close()
```

Comment:
Could use a comment with the motivation:

> The following test assertions use reflection to deep-equals the session objects. The session manager's `expireLoop` might be writing to those fields while reflection is reading those fields. Thus we close the session manager before proceeding to avoid a data race.

Reply:
updated